securedrop-client/debian/rules

```diff
@@ -4,7 +4,7 @@
 	dh $@ --with python-virtualenv
 
 override_dh_virtualenv:
-	dh_virtualenv --python /usr/bin/python3.5 --setuptools -S --index-url https://dev-bin.ops.securedrop.org/simple --requirements build-requirements.txt
+	dh_virtualenv --python /usr/bin/python3.7 --setuptools -S --index-url https://dev-bin.ops.securedrop.org/simple --requirements build-requirements.txt
```

Comment:
this will break stretch builds, so let's make this python3, also there's another hardcoded python3.5 in `securedrop-client.triggers` so we'll need to update there too